fastly/domains/v1/api_test.go

```diff
@@ -0,0 +1,125 @@
+package v1
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/fastly/go-fastly/v9/fastly"
+)
+
+func TestClient_Domain(t *testing.T) {
+	t.Parallel()
+
+	var err error
+	fqdn := "fastly-sdk-gofastly-testing.com"
+
+	// Create
+	var d *Data
+	fastly.Record(t, "create", func(c *fastly.Client) {
+		d, err = Create(c, &CreateInput{
+			FQDN: fastly.ToPointer(fqdn),
+		})
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
+	// Ensure deleted
+	defer func() {
+		fastly.Record(t, "cleanup", func(c *fastly.Client) {
```

Comment:
I don't think this one needs to be recorded, as there's an explicit test for the 'Delete' operation already.

Reply:
Done https://github.com/fastly/go-fastly/pull/577/commits/53475a3ba51f8e0768bf6597ef599ea894c9075b